AmplifyPlugins/Internal/Tests/InternalAWSPinpointUnitTests/ActivityTrackerTests.swift

```diff
@@ -79,7 +82,7 @@ class ActivityTrackerTests: XCTestCase {
         
         NotificationCenter.default.post(Notification(name: Self.applicationDidMoveToBackgroundNotification))
         NotificationCenter.default.post(Notification(name: Self.applicationWillMoveToForegoundNotification))
-        NotificationCenter.default.post(Notification(name: Self.applicationWillTerminateNotification))
+        await NotificationCenter.default.post(Notification(name: Self.applicationWillTerminateNotification))
```

Comment:
🤔 is the `await` really only needed for this particular notification?

Reply:
Yes, it's only one notification.  It's possible that this could change when Swift 6 is officially released.